server/src/internalClusterTest/java/org/elasticsearch/snapshots/GetSnapshotsIT.java

```diff
@@ -578,6 +581,26 @@ public void testSortAfter() throws Exception {
         assertThat(paginatedResponse2.totalCount(), is(3));
     }
 
+    public void testRetrievingSnapshotsWhenRepositoryIsMissing() throws Exception {
+        final String repoName = "test-repo";
+        final Path repoPath = randomRepoPath();
+        createRepository(repoName, "fs", repoPath);
+        final String missingRepoName = "missing";
+        createRepository(missingRepoName, "fs");
```

Comment:
Do we need to even create this repo only to delete it later? I would expect the test to work without that.

Reply:
Not really, I had the SLM issue reproduction path in mind, which is not necessary here :). I will clean it up.